analysis_options.yaml

```diff
@@ -2,6 +2,7 @@ analyzer:
   strong-mode: true
   language:
     enableGenericMethods: true
+    enableSuperMixins: true
```

Comment:
Are we using this feature in the dartdoc codebase? I'm curious why you're turning it on here...

Reply:
#1541 -- A flutter user tried to do this.  Flutter could be using it in their codebase -- it is enabled in their analysis options -- but I guess they just haven't yet or otherwise dartdoc wouldn't have worked for them.

Reply:
Oh, nvm, you mean why in analysis_options.yaml.  The test code has this now.